compiler_tester/src/lib.rs

```diff
@@ -143,12 +148,17 @@ impl CompilerTester {
         let _: Vec<()> = tests
             .into_par_iter()
             .map(|(test, compiler, mode)| {
+                let mode_string = mode.clone().to_string();
```

Comment:
Can be without `.clone()` I think.
Just in case, please run `cargo fmt && cargo clippy` before every commit.
We'll add it to CI soon so it's always verified.